### ISSUE TITLE
Add doc explaining the INSTALLED_APPS order importance

### ISSUE THREAD

**Issue by troygrosfield** [henriquebastos/django-test-without-migrations#3]
I couldn't find doc saying that this app needed to be included in the INSTALLED_APPS **before** the app that contains the actual management test command or else this app isn't picked up which is important to note.

Nice simple app that relieves a lot of headaches!  Thanks for releasing!

**Comment by henriquebastos** [comment]
Since the main `test` command is defined by *django's core* you shouldn't have had any trouble with that.

Are you using another 3rd party app that also overrides the `test` command?

**Comment by troygrosfield** [comment]
Not using any other 3rd party command that override **test**.  All I know is when I add it to the installed apps after the django apps it gives me the following error:
```
$ python manage.py test --nomigrations
usage: manage.py test [-h] [--version] [-v {0,1,2,3}] [--settings SETTINGS]
                      [--pythonpath PYTHONPATH] [--traceback] [--no-color]
                      [--noinput] [--failfast] [--testrunner TESTRUNNER]
                      [--liveserver LIVESERVER] [-t TOP_LEVEL] [-p PATTERN]
                      [-k] [-r] [-d]
                      [test_label [test_label ...]]
manage.py test: error: unrecognized arguments: --nomigrations
```
If I make it the first app installed apps, I don't receive the error.

**Comment by emanuelcds** [comment]
It does not work with django-nose and DjangoNoseTestRunner.

**Comment by henriquebastos** [comment]
Plz, open a new issue for this.

**Comment by alecklandgraf** [comment]
:+1: on the `INSTALLED_APPS` ordering

**Comment by mrmachine** [comment]
This should be fixed in #6

**Comment by henriquebastos** [comment]
Fixed. Thx all!
